atomic_reactor/plugins/pre_change_from_in_df.py

```diff
@@ -1,5 +1,5 @@
 """
-Copyright (c) 2015-2018 Red Hat, Inc
+Copyright (c) 2022 Red Hat, Inc
```

Comment:
Just a question, why change it to "2022" but not "2015-2022"?

Reply:
Updated.